### PR TITLE
Fetch Flaws with sorting

### PR DESCRIPTION
**Pull request by tboyetteredhat**
Fetch Flaws with sorting

### CODE REVIEW

File: src/components/IssueQueue.vue [hunk 1]
```diff
@@ -137,18 +134,17 @@ function handleScroll() {
 
   const totalHeight = tableContainerEl.value.scrollHeight;
   const scrollPosition = tableContainerEl.value.scrollTop + tableContainerEl.value.clientHeight;
-
-  if (scrollPosition >= totalHeight) {
+  if (scrollPosition >= totalHeight - 5) {
```
Comment (superbuggy) [inline src/components/IssueQueue.vue]:
```suggestion
  if (scrollPosition >= totalHeight * 0.95) {
```

or

```suggestion
  if (scrollPosition >= totalHeight * 0.99) {
```

with a comment to explain it's to ensure load on scroll works

File: src/views/IndexView.vue [hunk 2]
```diff
@@ -9,38 +9,41 @@ const issues = ref<any[]>([]);
...
 
-function fetchFlaws(filters: any = {}) {
+function fetchFlaws(params: any = {}) {
   offset.value = 0;
-  isFinalPageFetched.value = false;
+  isFinalPageFetched.value = false;  
+  isLoading.value = true;
 
-  getFlaws(offset.value, 0, filters.value)
+  getFlaws(offset.value, 0, params.value)
```
Comment (superbuggy) [inline src/views/IndexView.vue]:
```suggestion
  getFlaws(offset.value, 100, params.value)
```

**Review by superbuggy — CHANGES_REQUESTED** [review]
Looks solid, but maybe change limit arg from 0 to 100?

**Review by C-Valen — APPROVED** [review]
LGTM 🔥

File: src/components/IssueQueue.vue [hunk 3]
```diff
@@ -41,18 +41,30 @@ const filteredStates = computed(() => {
...
+    paramsObj.filter = issueFilter.value;
+  }
+
+  const sortOrderPrefix = isSortedByAscending.value ? '' : '-';
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  //@ts-expect-error
+  paramsObj.order = {
+    id: `${sortOrderPrefix}cve_id,${sortOrderPrefix}uuid`,
+    state: `${sortOrderPrefix}workflow_state`,
+  }[selectedSortField.value] || `${sortOrderPrefix}${selectedSortField.value}`;
```
Comment (superbuggy) [inline src/components/IssueQueue.vue:65]:
```suggestion
  paramsObj.order = {
    id: `${sortOrderPrefix}cve_id,${sortOrderPrefix}uuid`,
    state: `${sortOrderPrefix}workflow_state`,
    [selectedSortField.value]: `${sortOrderPrefix}${selectedSortField.value}`
  }[selectedSortField.value];
```

**Review by superbuggy — CHANGES_REQUESTED** [review]
I rewrote that last suggestion in a way that doesn't make TypeScript complain

**Comment by superbuggy** [comment]
Nice catch! Looks fantastic

**Review by superbuggy — APPROVED** [review]

**Pull request merged**